src/full/Agda/TypeChecking/Primitive/Cubical/Glue.hs

```diff
@@ -196,7 +196,7 @@ doGlueKanOp (TranspOp psi u0) (IsFam (la, lb, bA, phi, bT, e)) tpos = do
       --        (unglue_u0 i0)
```

Comment:
It seems that in the comment for `a1` the constraint `(φ = i1)` has to be updated to `(ψ = i1)` as well.